package.json

```diff
@@ -36,7 +38,8 @@
     "mocha": "^8.1.1",
     "npm-run-all": "^4.1.5",
     "nyc": "^15.1.0",
-    "rollup": "^2.23.0"
+    "rollup": "^2.23.0",
+    "rollup-plugin-node-polyfills": "^0.2.1"
```

Comment:
Why do we need this?

Reply:
It was somehow needed when the `inherits` dependency was included. I'll double-check whether it's needed anymore or remove it.

Reply:
That's because `inherits` is in Node `utils` module. The only case now where we use Node module is `fs` in the tests.